app/controllers/api/v1/concerns/exception_handler.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Concerns
+      module ExceptionHandler
+        extend ActiveSupport::Concern
+
+        def render_error(code, msg)
+          render status: code, json: { error: msg }.to_json
```

Comment:
Render the `error.class` for a more detailed response.

Reply:
https://github.com/SpicyCookies/rails-interview-project/pull/5/commits/bf27ef240700ec497e71d1aa4e8368d9c2670798